### PR TITLE
Added most of the content

### PR DESCRIPTION
**Pull request by QATomB**

### CODE REVIEW

File: source/biggestthreats.php [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+		<div id="gradient"></div>
+		<?php include "header.html"; ?>
+		<main>
+			<div id="splash-image"><h1>Biggest Threats</h1></div>
+			<div id="thing-container">
+				<div id="1" class="message-container">
+					<div class="left-text text">
+						<h2>Deforestation</h2>
+						<p>because trees store carbon as they grow, cutting or burning down trees releases that carbon into the atmosphere. Farmers may cut down trees or clear land using fire to produce soya for animal feed, such as in the Amazon. In other parts of the world, natural forests are cleared for timber, mining or palm oil.</p>
+					</div><div class="right-image image">
```
Comment (lonewonder0) [inline source/biggestthreats.php:389]: This formatting is hard to read. Please space your labels out more clearly.

File: source/biggestthreats.php [hunk 2]
```diff
@@ -0,0 +1,455 @@
+<!DOCTYPE html>
+<html>
+	<link rel="stylesheet" href="colours.css">
+	<head>
+		<title>EcoWithYou - Biggest Threats</title>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+		<link rel="icon" type="image/x-icon" href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAKUAAAClCAMAAAAK9c3oAAABoVBMVEVHcEym0pah
```
Comment (lonewonder0) [inline source/biggestthreats.php:7]: Change this to a source reference.

**Review by lonewonder0 — CHANGES_REQUESTED** [review]
